cmake/redisearch_cflags.cmake

```diff
@@ -45,3 +45,5 @@ ENDIF()
 
 SET(RS_C_FLAGS "${RS_COMMON_FLAGS} -std=gnu99")
 SET(RS_CXX_FLAGS "${RS_COMMON_FLAGS} -fno-rtti -fno-exceptions -std=c++11")
+SET(RS_SO_FLAGS "-Wl,-Bsymbolic,-Bsymbolic-functions")
```

Comment:
The issue we had with geo and graph was when RS was used as a dependency. Not sure we want to limit to 'SO'.

Reply:
@ashtul This is a linker option, thus it does nothing for compilation commands. When .so file is generated (like, when building a standalone RediSearch, it takes effect. On RedisGraph builds, where RediSearch is build as .a, the RedisGraph Makefile takes care of linking.
We need to ensure RSCoordinator does the right thing too.